src/containers/Orientation.js

```diff
@@ -1,16 +1,29 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
+const isMobileDevice = () =>
+  typeof window.orientation !== "undefined" ||
```

Comment:
I'm afraid that `window.orientation` is not going to be our best friend since it's deprecated.

Reply:
Oh wow I wasn't aware of this, I'll look for an alternative approach 😅

Reply:
Would this work? 🤔 
https://developer.mozilla.org/en-US/docs/Web/API/Detecting_device_orientation

Reply:
👌 makes sense, I can make those changes. Do you have a minimum width in mind?

Reply:
Any advance on this?